utils/src/utils.ts

```diff
@@ -0,0 +1,11 @@
+const getReferenceDataType = (val: unknown): string => Object.prototype.toString.call(val)
+
+export const isString = (val: unknown): val is string => typeof val === 'string'
+export const isNumber = (val: unknown): val is number => typeof val === 'number'
```

Comment:
`isString`、`isNumber`、`isArray`、`isObject` 
I extracted part of the commonly used functions to the warehouse https://github.com/baiwusanyu-c/bwsy-utils
`import { isString, isNumbe, isArray, isObject  } from 'baiwusanyu-utils'`

Reply:
I think it would be better to add the type predicate `is` in your functions.